plugin-server/src/utils/db/utils.ts

```diff
@@ -18,7 +18,10 @@ import { status } from '../../utils/status'
 import { areMapsEqual, castTimestampOrNow } from '../../utils/utils'
 
 export function unparsePersonPartial(person: Partial<InternalPerson>): Partial<RawPerson> {
-    return { ...(person as BasePerson), ...(person.created_at ? { created_at: person.created_at.toISO() } : {}) }
+    return {
+        ...(person as BasePerson),
+        ...(person.created_at ? { created_at: person.created_at.toISO() ?? undefined } : {}),
```

Comment:
`toISO` could have been returning `null`, and we were expecting `string | undefined`

Reply:
Watch this be a subtly very important bug that the whole codebase relied on 😆